docs/data-sources/organization_user_group.md

```diff
@@ -24,5 +24,6 @@ The Organization User Groupe data source provides information about the existing
 
 - `create_time` (String) Time of creation
 - `description` (String) The organization user group description. This property cannot be changed, doing so forces recreation of the resource.
+- `group_id` (String) Organization user group ID
```

Comment:
I'm not intimately familiar with Terraform internals, so I'm not sure what the difference between is between this `group_id` and the `id` below?